packages/web-components/src/components/va-pagination/va-pagination.tsx

```diff
@@ -147,101 +260,182 @@ export class VaPagination {
     const previousAriaLabel = ariaLabelSuffix ? `Previous page ${ariaLabelSuffix}` : 'Previous page';
     const nextAriaLabel = ariaLabelSuffix ? `Next page ${ariaLabelSuffix}` : 'Next page';
     const lastPageAriaLabel = ariaLabelSuffix ? `Page ${pages} ${ariaLabelSuffix}` : `Page ${pages}`;
+    if (uswds) {
+      const pageNumbersToRender = this.pageNumbersUswds();
+      const itemClasses = classnames({
+        'usa-pagination__item': true,
+        'usa-pagination__page-no': true
+      });
+      const ellipsisClasses = classnames({
+        'usa-pagination__item': true,
+        'usa-pagination__overflow': true
+      });
+      const arrowClasses = classnames({
+        'usa-pagination__item': true,
+        'usa-pagination__arrow': true
+      });
+
+      const previousButton = page > 1
+        ? 
+        <Fragment>
+          <li class={arrowClasses} aria-label={previousAriaLabel}>
+            <a class="usa-pagination__link usa-pagination__previous-page" href="javascript:void(0)">
+              <div id="previous-arrow-icon"></div>
+              <span class="usa-pagination__link-text">Previous</span>  
+            </a>
+          </li>
+          {!pageNumbersToRender.includes(1) && 
+          <Fragment>
+            <li class={itemClasses}>
+              <a href="javascript:void(0)" class="usa-pagination__button">1</a>
+            </li>
+            <li class={ellipsisClasses} aria-label="ellipsis indicating non-visible pages" role="presentation">
+              <span>...</span>
+            </li> 
+          </Fragment>}
+        </Fragment>
+        : null;
+      
+      const renderPages = pageNumbersToRender.map(pageNumber => {
+        const anchorClasses = classnames({
+          'usa-pagination__button': true,
+          'usa-current': page === pageNumber
+        })
 
-    const renderPages = this.pageNumbers().map(pageNumber => {
-      const pageClass = classnames({
-        'button-active': page === pageNumber,
-        'button-inner': true,
+        return (
+          <li class={itemClasses}>
+            <a href="javascript:void(0)" class={anchorClasses}>{pageNumber}</a>
+          </li>
+        )
       });
 
-      const pageAriaLabel = ariaLabelSuffix ? `Page ${pageNumber} ${ariaLabelSuffix}` : `Page ${pageNumber}`;
+      const nextButton = page < pages
+        ?
+        <Fragment>
+          {pages > VaPagination.SHOW_ALL_PAGES &&
```

Comment:
I'm not sure I understand why you need to use the class name `VaPagination` instead of just `this`

Reply:
I was using a static variable but changed it to instance variable to avoid confusion